packaging/examples/metrics/kafka-mirror-maker-2-metrics.yaml

```
@@ -203,12 +203,13 @@ data:
       type: GAUGE
 
     #kafka.connect:type=MirrorSourceConnector
-    - pattern: kafka.connect.mirror<type=MirrorSourceConnector, target=(.+), topic=(.+), partition=(.+)><>([a-z-_]+)
+    - pattern: kafka.connect.mirror<type=MirrorSourceConnector, source=(.+), target=(.+), topic=(.+), partition=(.+)><>([a-z-_]+)
       name: kafka_connect_mirror_mirrorsourceconnector_$4
```

You will need to update this as well -> you need to bump it to `$5`

```suggestion
      name: kafka_connect_mirror_mirrorsourceconnector_$5
```

---

good catch. i updated the example out put in the PR description with the corrected metric names after testing this.